### PR TITLE
Speakers Cards & Page Header

### PR DESCRIPTION
**Pull request by jeremiebardon**
- Adding a tailwind simple plugin for aspect Ratio
- Creating CSS for Speakers Card
- Creating CSS for Page Header

### CODE REVIEW

**Comment by f3ltron** [comment]
first thank's for your work ! 👍 

Can you please pull vuemontreal-next update on your branch ? 

I have some points i would like to add:
- is it possible to test if the video work or not ? Apparently i can't watch the youtube video maybe because its local ?
- i see the use of tailwind aspect ratio. Is it hard to do it by ourself ?
- if the video not work or doesnt exist what can we do. What do you think ?
- can you add a skeleton as for the home page

ps: I will update storyblok so you will have all the time good data and full so you will be able to work with good datas

**Comment by f3ltron** [comment]
i would like to limitate the size of the image of speakers. We can do it manually or we can use something form storyblok.

https://www.storyblok.com/docs/image-service we just need to re write the url for images

**Comment by aurelienroux** [comment]
I think we should also add **v-if** conditions on all different blocks. From what I saw, not every speaker has a description. This way, we don't generate empty blocks of markup with no data inside

```
<div v-if="speaker.description" class="flex flex-col w-full ml-4">
  <p>Description {{ speaker.description }}</p>
</div>
```

nice job 👍

File: pages/events/_eventId.vue [hunk 1]
```diff
@@ -2,46 +2,85 @@
   <!--  eslint-disable vue/valid-template-root  -->
   <div v-if="$fetchState.pending">loading</div>
   <article v-else>
-    <div>title event: {{ story.content.title }}</div>
-    <hr />
+    <h1 class="text-2xl font-bold leading-tight uppercase mb-2">
```
Comment (aurelienroux) [inline pages/events/_eventId.vue]: following the templates, event titles should not be uppercase

**Comment by f3ltron** [comment]
> I think we should also add **v-if** conditions on all different blocks. From what I saw, not every speaker has a description. This way, we don't generate empty blocks of markup with no data inside
> 
> ```
> <div v-if="speaker.description" class="flex flex-col w-full ml-4">
>   <p>Description {{ speaker.description }}</p>
> </div>
> ```
> 
> nice job 👍

same for gallery we dont have images for all events for now

**Comment by jeremiebardon** [comment]
Ok then I take in account these modifications and push again. 

- About the aspect ratio it's easy to do by our own, it was just for following the tailwindcss guideline (avoiding custom css and let the framework generate the custom class).

- We can hide video, description, gallery and social if they doesn't exist.

- What do you mean by adding skeleton for the home page ?

> i would like to limitate the size of the image of speakers. We can do it manually or we can use something form storyblok.
> 
> https://www.storyblok.com/docs/image-service we just need to re write the url for images

Ok for me.

Concerning the uppercase: I really don't know why I've add that (Miss the figma design). 

**RECAP:**

- If video, description, gallery and social missing. Hide them with conditionnal.
- Use https://www.storyblok.com/docs/image-service for liniting speakers image size
- Delete title uppercase

Is it ok for you ?

**Comment by f3ltron** [comment]
skeleton loading as i did for home page to have a nice preload data and no repaint.
I used it on the home page

And all other points it's ok for me yep

Don't hesitate to mp on slack i will add you on dev channel

**Comment by f3ltron** [comment]
rich text should use event-description component too becasue it will be a specific v-html

**Comment by aurelienroux** [comment]
@jeremiebardon here is an example of Skeleton loading
https://kitwon.github.io/vue-loading-skeleton/?path=/story/skeleton-component--default

it will create grey markup placeholders while your data finishes loading. you have an example on the index page

**Comment by jeremiebardon** [comment]
About the video: 
When you uploading video url from Youtube it's needed to add this format: https://www.youtube.com/**embed**/IBAR1MGhyaI. (The embeded link)

**Comment by f3ltron** [comment]
I ma not sure to understand for the youtube video. But you did a great job so i will accept it and create two new tickets / bugs for video and gallery

**Comment by jeremiebardon** [comment]
In fact the actual video url (exemple from  **004 - Inner workings of a JavaScript Framework**) is: https://www.youtube.com/watch?v=ayOhoNbOFtU&feature=youtu.be.

It's need to be: https://www.youtube.com/embed/ayOhoNbOFtU

**Comment by f3ltron** [comment]
ok i see i will update it. If you can have a look at my pr

**Pull request merged**